custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -144,14 +145,14 @@ class SummaryReportingRates(ReportingRatesData):
     use_datatables = True
 
     @property
+    @memoized
     def get_locations(self):
-        location_types = [
-            location_type.name
-            for location_type in Domain.get_by_name(self.domain).location_types
-            if location_type.administrative
-        ]
-        return SQLLocation.objects.filter(parent__location_id=self.config['location_id'],
-                                          location_type__name__in=location_types, is_archived=False)
```

Comment:
I assume this is also a relic from when location types were stored on the domain object.  Now this can be done much more simply.